.gitignore

```diff
@@ -1,3 +1,8 @@
+/venv/
```

Comment:
Replace with the default python gitignore: https://github.com/github/gitignore/blob/main/Python.gitignore